requirements.txt

```diff
@@ -7,3 +7,6 @@ XlsxWriter==1.3.8
 h5py==3.2.1
 pupynere-pdp==2.0.1
 requests==2.25.1
+numpy~=1.20
+gdal~=3.0
```

Comment:
From what I see on pypi `3.0.4` was the last of the `3.0`s. Is there anything stopping us from pinning `3.0.4` here?